config/errors.go

```diff
@@ -226,15 +226,44 @@ func (err TerragruntOutputListEncodingError) Error() string {
 }
 
 type TerragruntOutputTargetNoOutputs struct {
+	targetName    string
+	targetPath    string
 	targetConfig  string
 	currentConfig string
 }
 
+func (err TerragruntOutputTargetNoOutputs) ExitCode() int {
+	return 2
```

Comment:
Why 2?

Reply:
It's always hard to choose the correct exit code, but over the years I have typically used `2` for `Incorrect usage, such as invalid options or missing arguments`.

Happy to change it, just not sure if it should be `1`.

Reply:
Ah, let's leave it at 1 for now. 

A concern I have is that a user might be using `-detailed-exitcode` and mistakenly think that they have a successful plan with a non-empty diff:
https://opentofu.org/docs/cli/commands/plan/#other-options

I think it's safer to leave exit codes as 1 unless we specifically want to support adding tooling that detects unique exit codes for whatever reason.